logic.py

```diff
@@ -46,3 +66,18 @@ def wordcreator(word,partofspeech): # this function takes word selected and part
         return word
     else:
         print("Not Found")
+
+def SQL_log(statment,status,purpose):
+    now = datetime.now()
+    row = [statment, now.strftime("%d/%m/%Y, %H:%M:%S"),status,purpose]
+    headers = ["SQL Statment","Date & Time","Status","Purpose"]
+    if os.path.isfile('SQL-Scripts/sql_log.csv'):
+        with open('SQL-Scripts/sql_log.csv', 'a') as csvfile:  
```

Comment:
Are we reading sql_log file or sql_log_test csv file here?

Reply:
I think it is the sql_log file.